tests/cpp/test_tutorial.cpp

```diff
@@ -1250,4 +1250,204 @@ TEST_F(Tutorial, BasicTMA) {
   }
 }
 
+TEST_F(Tutorial, PointwiseBroadcastTMA) {
```

Comment:
Could you add a `NVFUSER_TEST_CUDA_ARCH_GUARD(9, 0);` to both tests, and move this test below `VectorizeStorePointwiseTMA`?